cmd/tke-installer/app/installer/types/types.go

```diff
@@ -32,6 +32,13 @@ type CreateClusterPara struct {
 	Config  Config     `json:"Config"`
 }
 
+func (c *CreateClusterPara) RegistryIP() string {
+	if c.Config.HA != nil {
+		return c.Config.HA.VIP()
```

Comment:
Use VIP have some problem, for local storage case, data of registry is in master0 node, but VIP backend could be master0, master1, master2....

So RegistryIP shouldn't use VIP.

Reply:
Currently we visit registy-api through tke-gateway, which is a daemonset. Based on this thought I think we could reach registry-api through any gateway running in the cluster, not only the one in master0. 
If there're some case we should visit tke-registry directly, please let me know.

Reply:
You are right. I forget tke-gateway listen 80/443 for registry.